### ISSUE TITLE
Can't build CanaryMod

### ISSUE THREAD

**Issue by walterhiggins** [CanaryModTeam/CanaryMod#232]
CanaryMod (and CanaryLib) cannot be built because repo.visualillusionsent.net is down.

```
[ERROR] Failed to execute goal on project CanaryMod: 
 Could not resolve dependencies for project net.canarymod:CanaryMod:jar:1.8.0-1.2.1-SNAPSHOT:
   Failed to collect dependencies for [net.canarymod:CanaryLib:jar:1.2.1-SNAPSHOT (compile), net.minecraft:server:jar:1.8.0_01-SNAPSHOT (compile), junit:junit:jar:4.12 (test), jline:jline:jar:2.11 (compile)]:
   Failed to read artifact descriptor for net.canarymod:CanaryLib:jar:1.2.1-SNAPSHOT: Could not transfer artifact net.canarymod:CanaryLib:pom:1.2.1-SNAPSHOT from/to vi-repo (http://repo.visualillusionsent.net/repository/public/): 
   Failed to transfer file: http://repo.visualillusionsent.net/repository/public/net/canarymod/CanaryLib/1.2.1-SNAPSHOT/CanaryLib-1.2.1-SNAPSHOT.pom. 
   Return code is: 503 , ReasonPhrase:Service Unavailable. -> [Help 1]
```

**Comment by jamierocks** [comment]
nexus.visualillusionsent.net

**Comment by walterhiggins** [comment]
Hi @jamierocks 
Can you tell me what changes you made to the pom.xml file to get this to build?

**Comment by jamierocks** [comment]
See the modifications made to pom.xml in https://github.com/CanaryModTeam/CanaryLib/commit/454154f02b4f80f51453bb4ca0d02ebf3946fd6b

**Comment by jamierocks** [comment]
I personally do not use CanaryMod, as I am working on my own implementation.

**Comment by walterhiggins** [comment]
So I need to host the javadocs for canarymod on my own site now ( http://scriptcraftjs.org/ ) as I have a book which includes references to the now-defunct canarymod-hosted javadocs. I was hoping a `mvn site` and ftp would suffice but the vi repo is also down so mvn won't build. Anyone else tried building CanaryMod / CanaryLib lately?

**Comment by jamierocks** [comment]
As I said, make the same modification made to pom.xml in https://github.com/CanaryModTeam/CanaryLib/commit/454154f02b4f80f51453bb4ca0d02ebf3946fd6b to CanaryMod.

TLDR; they changed their repo to nexus, and also the subdomain they used.

**Comment by jamierocks** [comment]
Also @Larry1123 has hosted javadocs.

They are usually https://docs.larry1123.net/ but they don't seem to be working currently. :/

**Comment by jamierocks** [comment]
@walterhiggins what do you mean the javadocs are down?

I can see them.

https://ci.visualillusionsent.net/job/CanaryLib/javadoc/

**Comment by Larry1123** [comment]
My docs site has not been working for some time, when I moved where my site is hosted I never got a system down to be able to host docs, been meaning to make some auto java thing to do it but never got around to it.

**Comment by iszilagyigit** [comment]
The link (http://nexus.visualillusionsent.net/content/groups/public/net/minecraft/server/1.8.0_01-SNAPSHOT/)  to build the CanaryLib. Even if the listing is ok the jar download is not working.
The 1.7.2D for example is ok.
![bildschirmfoto 2015-12-30 um 11 16 47](https://cloud.githubusercontent.com/assets/10026874/12049405/fb19eb78-aee6-11e5-84a6-12223af80b28.png)
![bildschirmfoto 2015-12-30 um 11 18 34](https://cloud.githubusercontent.com/assets/10026874/12049419/1ca6c806-aee7-11e5-9fe1-56b0c94f6985.png)

**Comment by Larry1123** [comment]
Here use this if you want http://public.repo.larry1123.net/
If it is not working let me know and I will make sure the version your looking for is there.

**Comment by darkdiplomat** [comment]
offical repository updated, jenkins is building again
